index.html

```diff
@@ -829,6 +831,70 @@ <h2><code>application/vc+ld+sd-jwt</code></h2>
       </table>
     </section>
   </section>
+  
+
+  <section id="json-web-token-claim-registrations">
+    <h3>JSON Web Token Claims</h3>
+    <p>
+      This section retained deprecated registered claim names, 
+      that have been submitted to the Internet Engineering Steering Group
+      (IESG) for review, approval, and registration with IANA in the
+      "JSON Web Token Claims Registry".
```

Comment:
As nearby, it seems there should be a link (direct or reference) to the "JSON Web Token Claims Registry"
```suggestion
      This section retains deprecated registered claim names, 
      that were previously submitted to the Internet Engineering Steering
      Group (IESG) for review, approval, and registration with IANA in the
      "JSON Web Token Claims Registry".
```

Reply:
I probably should not have merged this, given this PR is attempting to implement a WG resolution... its possible I will need to manually reimplement this change depending on what happens next.

Reply:
Indeed. As i said in my comment: as soon as we start down the road of the change, everything else should remain frozen...